src/utils/generate-vdm-util.ts

```diff
@@ -0,0 +1,68 @@
+/*!
+ * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+import { flags } from '@oclif/command';
+import { AlphabetLowercase, AlphabetUppercase } from '@oclif/parser/lib/alphabet';
+import { IBooleanFlag, IOptionFlag } from '@oclif/parser/lib/flags';
+import { GeneratorOptions } from '@sap/cloud-sdk-generator';
+import * as path from 'path';
+import { Options } from 'yargs';
+import { AllKeysTest, FlagsParsed, generatorOptionCLI } from '../commands/generate-vdm';
+
+export function toGeneratorSDK(flags: FlagsParsed): GeneratorOptions {
+  const result: Partial<GeneratorOptions> = {};
+  Object.keys(flags).forEach((key: string) => {
+    const keyTyped = key as AllKeysTest;
+    const value = flags[keyTyped];
+
+    if (value !== undefined && !generatorOptionCLI.hasOwnProperty(keyTyped)) {
+      if (typeof value === 'boolean' && keyTyped) {
+        (result[keyTyped as keyof GeneratorOptions] as boolean) = value;
+      }
+      if (typeof value === 'string') {
+        (result[keyTyped as keyof GeneratorOptions] as string) = value;
+      }
+    }
+  });
+  return result as GeneratorOptions;
+}
+
+export function toBoolean(yargsBool: Options | undefined): IBooleanFlag<boolean> {
```

Comment:
```suggestion
export function toBoolean(yargsBool?: Options): IBooleanFlag<boolean> {
```

Also is it necessary to put the return type explicitly? I would assume it can be inferred from the `return flags.boolean()`, right?

Reply:
see above